docs/project/2024/gabriel.danaila/index.md

```diff
@@ -0,0 +1,73 @@
+# DIY Pico Modulated Beeps to Tesla Coil
+This Tesla Coil has been modified so that the spark "sings" melody generated by a Raspberry Pico Microcontroller.
+
+:::info 
+
+**Author**: Gabriel Dănăilă \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-Gabi10017
+
+:::
+
+## Description
+
+The audio beeps are generated on the microcontroller which uses PWM in order to send a digital signal to a DAC which converts the signal into an analog one and then sends it to the Tesla Coil's PCB which modulates it into electrical voltage which is sent to the coil, playing a sound. The beep's pitch and volume can be controlled using 2 potentiometers connected using ADC.
+
+## Motivation
+
+I chose this project because I think it is an interesting, innovative idea to put in practice, that is neither too easy or too complex to do for a single-person project. This project idea was brought to me by the lab teacher.
+
+## Architecture 
+
+Add here the schematics with the architecture of your project. Make sure to include:
+ - what are the main components (architecture components, not hardware components)
+ - how they connect with each other
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+Detail in a few words the hardware used.
+
+### Schematics
+
+Place your KiCAD schematics here.
+
+### Bill of Materials
+
+<!-- Fill out this table with all the hardware components that you might need.
+
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
+
```

Comment:
Remove these libraries if you dont need them.

Reply:
Removed.